invokeai/frontend/web/src/app/components/InvokeAIUI.tsx

```diff
@@ -44,6 +44,7 @@ interface Props extends PropsWithChildren {
     imageName: string;
     action: 'sendToImg2Img' | 'sendToCanvas' | 'useAllParameters';
   };
+  selectedWorkflow?: string;
```

Comment:
Is this the actual workflow or the ID?

Reply:
The ID from WorkflowLibrary.workflow_id

Reply:
I would make this `selectedWorkflowId` in that case